tasks/eva_3399/src/main/groovy/eva3399/BackPropagateUpdatedRS.groovy

```diff
@@ -25,12 +25,15 @@ class BackPropagateUpdatedRS {
         def backPropagatedRSWriter =
                 this.backPropEnv.springApplicationContext.getBean(BeanNames.BACK_PROPAGATED_RS_WRITER,
                         BackPropagatedRSWriter.class)
-        def opsOfImpactedSS = [svoeClass, dbsnpSvoeClass].collect{new RetryableBatchingCursor<>(
-                where("_id").regex("EVA3399_UPD_LOCUS_${this.remappedAssembly}_.*").and(
-                        "inactiveObjects.remappedFrom").is(this.sourceAssembly),
-                this.backPropEnv.mongoTemplate, it)}
-        opsOfImpactedSS.each{it.each{opsInBatch ->
-            def impactedSSIds = opsInBatch.collect{it.accession}.toSet()
+        def impactedSS = [sveClass, dbsnpSveClass].collect{new RetryableBatchingCursor<>(
```

Comment:
Yes, that is correct. The reason I changed this was because of a remapped variant normalized from asm1 might not need its locus updated. Because the remapped locus agrees with the SS locus  in the database.

Given **asm1,ss1,ssUnnormalLocus1** and **asm2,ss1,ssLocus2**:

asm1, ss1, ssUnnormalLocus1 **---normalized to-->**  asm1, ss1, ssNormalizedLocus1 **---remapped to-->** asm2,ss1, ssLocus2 (agrees with existing ss locus in asm2)